addons/main/script_mod.hpp

```diff
@@ -9,8 +9,8 @@
 #define VERSION_AR MAJOR,MINOR,PATCHLVL,BUILD
 
 // MINIMAL required version for the Mod. Components can specify others..
-#define REQUIRED_VERSION 1.76
-#define REQUIRED_CBA_VERSION {3,4,1}
+#define REQUIRED_VERSION 1.80
```

Comment:
Doing 1.78 here is fine I think, maybe Linux port reaches 1.78 and not 1.80 for whatever reason. :P